easybuild/easyconfigs/a/Autoconf/Autoconf-2.69-GCCcore-6.4.0.eb

```diff
@@ -18,7 +18,13 @@ toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
 
 source_urls = [GNU_SOURCE]
 sources = [SOURCELOWER_TAR_GZ]
-checksums = ['954bd69b391edc12d6a4a51a2dd1476543da5c6bbf05a95b59dc0dd6fd4c2969']
+
+patches = ['Autoconf-%(version)s_config.guess_ppc64le.patch']
+
```

Comment:
@omula Style nitpicking: please drop the empty line above/below the `patches =` line, we usually keep `source_urls`, `sources`, `patches` and `checksums` "glued" together